node-repository/pom.xml

```diff
@@ -77,6 +77,11 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk15on</artifactId>
+            <scope>provided</scope>
```

Comment:
This must be compile scope - the bouncycastle bundles are not provided jdisc/configserver.